Scrape-Subreddit-Users-master/GetSubredditSubscribersComments.py

```diff
@@ -26,7 +27,7 @@
 print("Time elapsed to scrape %d posts: %d seconds" % (_nposts, int(end-start)))
 print("# Unique subreddit users found: %d" % len(s.authors))
 
-session = Session()
+session = Session() # Aaaaaand this -- RMD
```

Comment:
Like this. This dude right here. Riddle me this: 

Where is this session being defined?